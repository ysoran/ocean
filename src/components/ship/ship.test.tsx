import React from 'react';
import { shallow } from 'enzyme';
import Ship from './ship';

const setup = () => {
    const wrapper = shallow(<Ship lat={12} lng={12} idle={true} dest={""} eta={0} />);
    return {
        wrapper,
    };
};

describe('Ship component', () => {
    it('should render snapshot', () => {
        const { wrapper } = setup();
        expect(wrapper).toMatchSnapshot();
    });
});
