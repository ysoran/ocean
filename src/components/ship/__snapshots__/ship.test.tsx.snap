// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ship component should render snapshot 1`] = `ShallowWrapper {}`;
